Tutorials/bash/r-shiny-server/dx_shiny/src/dx_shiny.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# dxshiny 0.0.1
+
+set -eux
+
+main() {
+
+    # Download the input: your Shiny app's source code archive from the project
+    echo "Value of app_gz: '$app_gz'"
+    dx download "$app_gz" -o app.gz
+
+    # Unpack the app code. Here, `--strip 1` allows to skip the top folder
+    # of the archive and unpack the code directly underneath ./app/
+    # This allows you to go to the job's URL and launch your Shiny app directly.
+    # Otherwise, the code would be one folder level deeper and one would have to
+    # click on the app folder's link in the browser.
+    mkdir app
+    tar -zxvf app.gz -C app --strip 1
```

Comment:
should this be `tar.gz` ?

Reply:
It is downloaded and saved as 'app.gz' and therefore I am using it here too. It does not really matter though.

Reply:
@xquek , actually, it depends on how you run a command. If you run `tar -zcvf app.gz app_dir` it will create `app.gz` without adding additional `.tar.`. So, it is a convention rather than a rule. I am for shorter file names :-)